modules/ttdBidAdapter.js

```diff
@@ -125,14 +125,14 @@ function getUser(bidderRequest) {
 function getSite(bidderRequest, firstPartyData) {
   var site = {
     id: utils.deepAccess(bidderRequest, 'bids.0.params.siteId'),
-    page: utils.deepAccess(bidderRequest, 'refererInfo.referer'),
+    page: utils.deepAccess(bidderRequest, 'refererInfo.page'),
     publisher: {
       id: utils.deepAccess(bidderRequest, 'bids.0.params.publisherId'),
     },
     ...firstPartyData.site
   };
 
-  var publisherDomain = config.getConfig('publisherDomain');
+  var publisherDomain = bidderRequest.refererInfo.domain;
```

Comment:
might be helpful to rename this var so it doesnt show up in searches

Reply:
disagree, it's a game of whack-a-mole (there are many other instances of this name) and names should be chosen for clarity, not ease of grepping through unrelated code elsewhere.